### ISSUE TITLE
feature request: export XHThtml file that preview the same as renderd file

### ISSUE THREAD

**Issue by huangjj27** [simov/markdown-viewer#34]
I try finding a simple way export Readable document from markdown, until I meet this markdown viewer. However, the PDF file saved from chrome has some problem with styling. (for example, the margin line with page breakings seem not so beautiful, and some table's style also missed for some themes), so if there is a way to export a XHThtml file that get all js, images tables in a file will be great!

**Comment by simov** [comment]
Interesting idea, I'll think about implementing an `export` option.

**Comment by MarianoWalter** [comment]
Any news on this?

**Comment by huangjj27** [comment]
Nothing new，even havn’t gotten any feedback from author. It seems that simov has deperated this project?

**Comment by simov** [comment]
Yes, I don't have that feature ready yet, and no, I haven't abandoned the project. That being said, it's obvious that it's taking me way too much time to get to it.

**Comment by petko10** [comment]
+1 for exporting to HTML. I thought it would be possible to just view the source of the rendered page, but alas it seems it's not that simple, or I'm missing something.

**Comment by tianh4** [comment]
+1，Hope this feature can be implemented

**Comment by DebuggingMode** [comment]
+1, it will be great feature

**Comment by Lizxs** [comment]
+1

**Comment by aemonge** [comment]
+1

**Comment by Nebula-Developer** [comment]
Any news?

**Comment by simov** [comment]
Not yet, though it is definitely on my backlog. In last release I did some improvements around the PDF export, namely breaking the content between pages a bit more intelligently, so hopefully that works for at least some of the cases. A while ago I tried exporting into MHTML instead as there is an API for that, but the result was not great, and also a MHTML viewer is needed.

**Comment by fmmoser** [comment]
I would also be very happy about such an export function. Is there any news?

**Comment by rvernica** [comment]
A _Save as HTML_ feature would be really useful addition to this extension.

**Comment by huangjj27** [comment]
I've turned to mdbook and typst
